docs/templating.md

```diff
@@ -14,6 +14,10 @@ This is an overview of the templating features available in nunjucks.
 > lacking here. Read about the differences
 > [here](http://mozilla.github.io/nunjucks/faq.html#can-i-use-the-same-templates-between-nunjucks-and-jinja2-what-are-the-differences).
 
+## File Extensions
+
+Although you are free to use any file extension you wish for your Nunjucks template files the Nunjucks community has adopted  `.njk`.  If you are developing tools or editor syntax helpers for Nunjucks please include recognition of this extension.
```

Comment:
It was discussed do the mailing list. I said I thought it made sense to recommend something, and I didn't care between .nj, .njs, or .njk.